apps/condo/domains/common/utils/text.js

```diff
@@ -53,6 +53,19 @@ function normalizeText (text) {
         .join('\n')
 }
 
+/**
+ *
+ * @param {string} text
+ * @param {number} length
+ */
+function shortenText (text, length) {
```

Comment:
If that util used on client side I would say it is redundant. Most of components such as Typography already had `ellipsis` property for that

Reply:
removed it